app/models/workpackage/workpackage-serializers.ts

```diff
@@ -26,6 +26,7 @@ export class WorkpackageDeserializer extends RestModelDeserializer<Workpackage>
   ];
 
   createInstance(): Workpackage {
-    return new Workpackage();
+    var wp :Workpackage = new Workpackage();
```

Comment:
besides using `var`, which should not be used over `let` unless there is a reason to do so, I don't see why this is necessary?